recipes/r-rphylip/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = 'Rphylip' %}
+{% set version = '0.1-23' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-{{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: https://cran.r-project.org/src/contrib/{{ name }}_{{ version }}.tar.gz
+#    - https://cran.r-project.org/src/contrib/Archive/{{ name }}/{{ name }}_{{ version }}.tar.gz
+  md5: 99597a778fc5bf69c7175e4dde787970
+
+build:
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base >=3.3.1
```

Comment:
Pnning removed. Originally the recipe didn't build unless this was in place, but now it works all right. I suspect that I managed to do an update in beteen.